barcode/codex.py

```diff
@@ -258,5 +258,25 @@ def render(self, writer_options=None, text=None):
         return Barcode.render(self, options, text)
 
 
+class Gs1128(Code128):
```

Comment:
Would you mind renaming this to `GS1_128`? Otherwise it's slightly confusing and kinda looks like `GS 1128` at first glance.

Reply:
At first, I named it like this but then I've changed because it's not pep8 compliant. But I don't mind changing it again to `GS1_128`, readability is important.

Reply:
Done

Reply:
Oh, I'd forgotten that was not pep8-compliant. I usually prefer to stick to it quick closely, but, honestly, I think this is a pretty valid exception anyway.

Thanks!